content/docs/querying/functions.md

```diff
@@ -85,6 +95,13 @@ rate(http_requests_total{job="api-server"}[5m])
 
 `rate` should only be used with counters.
 
+## `round()`
+
+`round(v instant-vector, to_nearest=1 scalar)` rounds the sample values of all
+elements in `v` to the nearest integer. Ties are resolved by rounding up. The
+optional `to_nearest` argument allows specifying the nearest multiple to which
+the sample values should be rounded.
```

Comment:
Should we make it clearer that the multiple can be a fraction ?

Given that the text starts by stating that it the value gets rounded to the nearest _integer_, the reader might not get it works also with e.g. `0.1`.